### DISCUSSION TITLE
Does KeePassXC reference use the attributes in options when checked in a clone?

### DISCUSSION THREAD

**Discussion by GoodGuy98** [keepassxreboot/keepassxc#6829] [Q&A] [answered]
I really appreciate the ATTRIBUTES feature in Options, I created several password entries clones from the top one.
Does KeePassXC share the Attributes entries from the top one. I would want the Attributes cloned so it's not necessary to manually 
update it in the Cloned password entries.

**Comment by droidmonkey** [comment, marked as answer]
You can read all about cloning here: https://keepassxc.org/docs/KeePassXC_UserGuide.html#_clone_an_entry

Step 4 shows the reference syntax which you can use for other attributes

**Reply by GoodGuy98** [reply]
Thank you for the help.
